### DISCUSSION TITLE
Placing portable objects on the game map, plus usecode help

### DISCUSSION THREAD

**Discussion by r-h-o-n-a** [exult/exult#289] [General]
Hi Exulters :) Just wanting to ask a question that probably has a blindingly clear answer I'm missing. I'm trying to place new objects in my Serpent Isle mod; while some objects (walls, doors, etc) are happily staying put, I'm not sure how to use Exult Studio to place new portable objects onto the map and stay there without also messing up the process of starting a new game; once I've placed objects, when I quit and "Start New Game" Exult brings me back to the spot I was at when I was placing objects, not in the proper starting sequence. Has anyone else had this issue, and how did you get around it?

Also, I'm really struggling with working out how to get into the usecode - after much fiddling around with Exult Tools, I still can't work out the proper syntax to decompile the usecode for editing. Reinstalling Exult, Exult Tools, and even SI itself to ensure the paths are all correct hasn't helped either. Can y'all assist me with the process of getting at the usecode?

**Comment by KnightCaptainU7** [comment]
In SI the Avatar immediately triggers an egg which starts off the opening cutscene when you start the game normally.
There are some command line incantations needed to start the game in editor mode. Something like "exult.exe --edit" or similar?
Take note of the Avatar's position.
You'll need to move around by dragging the Avatar to the area you are working on, place items, etc.
Then use the F2 cheat menu to put the Avatar back at the original spot.
It's a good idea to keep track of the changes in a text file.

The Usecode extractor will output the code in human-readable form, but it will take a massive sed script to make it more readable. I have the laptop where I did this code handy, let me see if I can find the output file I had.

**Reply by r-h-o-n-a** [reply]
Much appreciated. I'll try the steps you mentioned and will update about how it goes.

I'm grateful also for anything you can do with regard to the usecode - that'd be great. (I know I emailed you about this a few days ago, but when I posted to the forum I wasn't meaning to passive-aggressively hurry you up or anything, just wanting to see if anyone else had additional insights. Sorry if it came across otherwise!)

**Reply by r-h-o-n-a** [reply]
Success! I tested your process by introducing the otherwise-unused cat skull into Fedabiblio's study and I think I've got a handle on it now. Thanks again.

![Catskull](https://user-images.githubusercontent.com/93699238/207233830-ff0f6849-26a6-48b6-982d-dfafac7ebdce.png)

**Comment by KnightCaptainU7** [comment]
On Windows, the shortcut to Exult can be copied and edited so the executable path ends with "C:\yourpath\Exult.exe" --edit
